Resources/Textures/Shaders/radial_blur.swsl

```diff
@@ -0,0 +1,17 @@
+﻿uniform sampler2D SCREEN_TEXTURE;
+uniform highp float VisualScale; // between 0 and 1
+const highp float Intensity = 0.2;
+const highp int SampleCount = 16; // multiple of 2
+
+// a simple radial blur
+void fragment() {
+    highp vec2 uv = FRAGCOORD.xy * SCREEN_PIXEL_SIZE.xy;
+    highp vec2 direction = vec2(0.5, 0.5) - uv;
```

Comment:
Good point, but I thought a little about it and decided to leave it as it is. Instead of restricting vision in a circle the idea this shader is to be psychologically close to the feeling of being drowsy by blurring your view "around the edges". So even if you have a widescreen resolution, the effect should not be stronger on the left or right side than on the top or bottom.